RELEASE/scripts/autoscend/auto_island_war.ash

```diff
@@ -271,38 +317,34 @@ boolean L12_startWar()
 		use_skill(1, $skill[Incredible Self-Esteem]);
 	}
 
-	autoAdv(1, $location[Wartime Hippy Camp]);
-	set_property("choiceAdventure142", "3");
-	if(contains_text(get_property("lastEncounter"), "Blockin\' Out the Scenery"))
+	// set noncombats to value needed to start the war
+	set_property("choiceAdventure142", "3");	//if wearing [Frat Warrior Fatigues] start the war or skip adventure
+	set_property("choiceAdventure146", "3");	//if wearing [War Hippy Fatigues] start the war or skip adventure
+
+	// wear the appropriate war outfit based on auto_hippyInstead
+	warOutfit(false);
+
+	// start the war when siding with frat boys
+	if(!get_property("auto_hippyInstead").to_boolean())
 	{
-		if(!get_property("auto_hippyInstead").to_boolean())
+		auto_log_info("Must save the ferret!!", "blue");
+		autoAdv(1, $location[Wartime Hippy Camp]);
+		
+		if(contains_text(get_property("lastEncounter"), "Blockin\' Out the Scenery"))

```

Comment:
I would change this condition to `internalQuestStatus("questL12War") == 1`
If the name of the encounter gets changed (not impossible, the game has been adding double spaces to certain stuff and fixing badly named things like gr8tness -> gr8ness recently) this will always be false.

Reply:
change made. for now

this function actually needs to be entirely replaced with a dedicated function that starts the arena for both sides. Because at the moment this is the only place where the arena sidequest is started, and only for the frat side, and I have seen it fail to start it for the frat before.